runtime/Java/src/org/antlr/v4/runtime/CharStreams.java

```diff
@@ -303,4 +303,24 @@ public static CodePointCharStream fromChannel(
 			channel.close();
 		}
 	}
+
+	/**
+	 * Takes the stream and forces all symbols to uppercase.
```

Comment:
perhaps say "Takes the stream and forces all symbols to uppercase for lexing purposes but leaves the original text as-is."?

Reply:
Done.